presto-tests/src/test/java/com/facebook/presto/memory/TestMemoryManager.java

```diff
@@ -126,12 +127,22 @@ public void testClusterPools()
                 }
             }
 
-            // Release the memory in the reserved pool
+            for (TestingPrestoServer server : queryRunner.getServers()) {
+                MemoryPool system = server.getLocalMemoryManager().getPool(SYSTEM_POOL);
+                assertTrue(system.getFreeBytes() < system.getMaxBytes());
```

Comment:
Shouldn't this check `getFreeBytes() < 0`? You manually reserved the entire pool, so this check will always pass